### PR TITLE
 modify item model's test code

### PR DESCRIPTION
**Pull request by hewonjeong**
- modify item model's test code
- modify mock data's keys following our key rule

### CODE REVIEW

File: src/test/node/items/models.js [hunk 1]
```diff
@@ -83,10 +83,18 @@ test('Check endTime value and change indexing items', t => {
...
+    return new Promise((resolve) => {
+      ItemManager.validChecker(expiredItem, (result)=>{
+        t.equal(result, expected.result, 'should be same result');
+        resolve();
+      });
+    });
+  })
+  .then(()=>{
+    return new Promise((resolve) => {
+      setTimeout(resolve, 2000);
```
Comment (ovekyc) [inline src/test/node/items/models.js:103]: Please leave comment why use this function

**Comment by ovekyc** [comment]
Please separate editing test code PR and 
editing mockdata PR next time

**Comment by ovekyc** [comment]
LGTM

**Comment by festinalente91** [comment]
LGTM

File: src/test/node/items/models.js [hunk 2]
```diff
@@ -83,10 +83,24 @@ test('Check endTime value and change indexing items', t => {
...
+  })
+  .then(()=>{
+    /* eslint-disable max-len */
+    /*
+      validChecker function implements changing state code of indexing item asynchronously if it's invalid.
+      this implementation is runned after callback to prevent perfomance issue in item get API so needs delay for test
+    */
+    /* eslint-enable */
+    return new Promise((resolve) => {
+      setTimeout(resolve, 2000);
```
Comment (festinalente91) [inline src/test/node/items/models.js:103]: seems tricky, but have no alternatives :)

**Comment by festinalente91** [comment]
LGTM

**Pull request merged**